pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -1416,6 +1420,8 @@ public void doNamespaceBundleSplit() throws Exception {
 
     @Override
     public void stop() throws PulsarServerException {
-        // do nothing
+        loadReportCacheZk.shutdown();
+        availableActiveBrokers.shutdown();
+        scheduler.shutdown();
```

Comment:
Also, one more concern:
1. Now, we have two `ZooKeeperChildrenCache` objects (one in each load-manager) and both use the same zk-session to get data from zk.
2. Now, while getting data from zk, we pass these objects as watch. So, we will set multiple watch for the same znode
3. and I think per zk-session we may get only 1 watch so, there could be possibility that new load-manager may not get watch ??
@merlimat